### DISCUSSION TITLE
我的巨亏经历 # 组合季报(2022Q3) · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#123] [Announcements]
# 我的巨亏经历 # 组合季报(2022Q3) · 构建我的被动收入

被动收入投资组合2022年Q3季报

https://www.bmpi.dev/money/passive-income-protfolio/2022q3/

**Comment by yongganyidian** [comment]
刚开始接触，向您学习

**Comment by icodingc** [comment]
有老飞行员和大胆的飞行员，但没有老的、大胆的飞行员。——E. Hamilton Lee
---
真正让你舒服的投资，才是好的投资。涨跌都在逻辑中
